modules/sriov-operator-metrics.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="sriov-operator-metrics_{context}"]
+= Enabling the SR-IOV Network Operator metrics
+
+// What are these metrics used for? 
+// Do we need description for each of the metrics?
+
+The SR-IOV Network Operator metrics are disabled by default. To enable the SR-IOV Network Operator metrics, you must set the `featureGates` field to `true`. 
+
+Following are the SR-IOV Network Operator metrics:
+
+* `sriov_vf_rx_broadcast`
+* `sriov_vf_rx_bytes`
+* `sriov_vf_rx_dropped`
+* `sriov_vf_rx_multicast`
+* `sriov_vf_rx_packets`
+* `sriov_vf_tx_bytes`
+* `sriov_vf_tx_dropped`
+* `sriov_vf_tx_packets`
+* `sriov_kubepoddevice`
+
+.Prerequisites
+
+* You have installed the {oc-first}.
+* You have logged in as a user with `cluster-admin` privileges.
+* You have installed the SR-IOV Network Operator.
+
+.Procedure
+
+* Set the `featureGates` field to `true` by running the following command:
++
+[source,terminal]
+----
+$ oc patch -n openshift-sriov-network-operator sriovoperatorconfig/default \
+    --type='json' -p='[{"op": "replace", "path": "/spec/featureGates/metricsExporter", "value":true}]'
```

Comment:
this does not work if the `.spec.featureGates` map is nil:

```suggestion
$ oc patch -n openshift-sriov-network-operator sriovoperatorconfig/default \
    --type='merge'  -p='{"spec": {"featureGates": {"metricsExporter": true}}}'
```

Reply:
Updated as per the suggestion.